### DISCUSSION TITLE
Better indication of modified/restrictive licensing

### DISCUSSION THREAD

**Discussion by 67372a** [civitai/civitai#104] [Ideas]
I noted at least three models have been posted with restrictive licensing that is mentioned in the description, often requiring that show more be clicked to even see it.

While users should be careful regarding use of models and their licenses, the UI doesn't really account for modified or custom licensing (that is usually more restrictive, not less), the models with custom or modified terms just have the standard creativeml-openrail-m set and put their additional terms in the description. 

I think it would be a good idea to highlight restrictive licensing better via clear indication on the model, tags, and perhaps even adding a filter. It may not be a bad idea to even implement an user agreement step.

**Comment by JustMaier** [comment]
We were actually talking about this earlier this week: [The plan](https://github.com/civitai/civitai/discussions/101)

Our plan is to make it easy for users to check boxes and automatically select a corresponding Creative Commons license and then display that inside the model details so it's clearer as to what is allowed with a given model.

**Comment by JustMaier** [comment]
**This is live!**
We opted to just display indicators instead of an actual Creative Commons license due to the uncertainty around copyright ownership and AI work.

**Setting model permissions**
![image](https://user-images.githubusercontent.com/607609/210704238-5c1ce32c-e1b7-42b3-adde-a8c04b5f0a4e.png)

**Viewing model permissions**
![image](https://user-images.githubusercontent.com/607609/210704167-06f2ffdd-f118-4538-98ef-cc009531243e.png)
